packages/client/package.json

```diff
@@ -40,11 +40,10 @@
 	"dependencies": {
 		"@overture-stack/lectern-dictionary": "workspace:^",
 		"@overture-stack/lectern-validation": "workspace:^",
+		"axios": "^1.7.2",
```

Comment:
comment.

web standard fetch is available in node
axios is overkill
aligns with removing things we don't need (`body-parser` removes in preference for native express api)

Reply:
`fetch` in any of its flavors doesn't include cancelling (you'd have to instantiate an AbortController separately), which is something we've been meaning to start implementing consistently.

does that add any nuance to this choice?